etcdserver/server.go

```diff
@@ -1435,20 +1435,20 @@ func (s *EtcdServer) TransferLeadership() error {
 		return nil
 	}
 
-	if !s.isMultiNode() {
```

Comment:
can we remove `isMultiNode` or just encapsulate `if s.cluster == nil || len(s.cluster.VotingMemberIDs()) <= 1 { ` in there?

Reply:
It is removed.